status/flowcells.py

```diff
@@ -244,7 +255,13 @@ class FlowcellLinksDataHandler(SafeHandler):
     def get(self, flowcell):
         self.set_header("Content-type", "application/json")
         try:
-            p = lims.get_containers(name=flowcell[8:])[0]
```

Comment:
@Galithil know you will hate me for this comment... but there is no way to abstract this check so we avoid this code repetition? would be much better